plugins/module_utils/data_set.py

```diff
@@ -321,6 +323,10 @@ def allocate_model_data_set(ds_name, model, vol=None):
             alloc_cmd = """{0} -
             VOLUME({1})""".format(alloc_cmd, vol.upper())
 
+        if asa_text:
+            alloc_cmd = """{0} -
+            RECFM(F,B,A)""".format(alloc_cmd)
```

Comment:
I thought that when a `dest` does not exists where `zos_copy` will create it, doesn't `zos_copy` also honor the volume option for destinations that are created by `zos_copy` , maybe I am remembering it incorrectly.

Reply:
It does honor the volume specified by the user. The record format option doesn't overwrite the volume one, it's appending to the string

Reply:
Ah, yup i see the append now, my bad and why I should not do late night reviews, thanks for explaining.